export_layers.py

```diff
@@ -10,77 +10,125 @@
 
 sys.path.append('/usr/share/inkscape/extensions')
 import inkex
+#inkex.localization.localize()
 
 Layer = collections.namedtuple('Layer', ['id', 'label', 'tag'])
 Export = collections.namedtuple('Export', ['visible_layers', 'file_name'])
 
-FIXED = '[fixed]'
+FIXED  = '[fixed]'
+F      = '[f]'
 EXPORT = '[export]'
+E      = '[e]'
 
-PNG = 'png'
+PDF = 'pdf'
 SVG = 'svg'
+PNG = 'png'
 JPEG = 'jpeg'
 
 
 class LayerExport(inkex.Effect):
     def __init__(self):
         inkex.Effect.__init__(self)
-        self.OptionParser.add_option('-o', '--output-dir',
+        self.arg_parser.add_argument('-o', '--output-source',
                                      action='store',
-                                     type='string',
-                                     dest='output_dir',
+                                     type=str,
+                                     dest='output_source',
```

Comment:
These are ArgParser defaults, may be omitted (I didn't know OptionParser, so I left these parameters previously). I'd rather add `required=True`.
Basically same for other `add_argument`.
I can perform some clean up later though, so it's up to you whether to apply the comment or not.

Reply:
Sure.